packages/react-app/src/constants.ts

```diff
@@ -166,7 +166,7 @@ export const ENUM = {
 
 // Default values
 
-export const DEFAULT_PAGE = ENUM_PAGES.List;
+export const DEFAULT_PAGE = undefined; // FIXME Hack fix to prevent load Dashboard
```

Comment:
TODO ? (more conventional to track) ,can be TODO : FIXME

Reply:
i'll change to TODO, but FIXME its kindy more critical for me :D

Reply:
Fixme